docs/tutorials/lightning_workflow.ipynb

```diff
@@ -0,0 +1,791 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c31688e6-98f1-4036-85b7-7116d11b42a4",
+   "metadata": {},
+   "source": [
+    "# Using Raster Vision with Lightning\n",
+    "\n",
+    "![The Lightning logo.](../img/lightning-logo.png \"The Lightning logo.\")\n",
+    "\n",
+    "[Lightning](https://www.pytorchlightning.ai/) (formerly known as PyTorch Lightning) is a high-level library for training PyTorch models. In this tutorial, we demonstrate a complete workflow for doing semantic segmentation on SpaceNet Vegas using a combination of Raster Vision and Lightning. We use Raster Vision for reading data, Lightning for training a model, and then Raster Vision again for making predictions and evaluations on whole scenes. \n",
+    "\n",
+    "Raster Vision has easy-to-use, built-in model training functionality implemented by the `Learner` class which is shown in the `train.ipynb` notebook. However, some users may prefer to use Lightning for training models, either because they already know how to use it, and like it, or because they desire more flexibility than the `Learner` class offers. This notebook shows how these libraries can be used together, but does not attempt to use either library in a particularly sophisticated manner."
```

Comment:
>which is shown in the `train.ipynb` notebook

It would be nice to link to that tutorial here.

https://nbsphinx.readthedocs.io/en/latest/markdown-cells.html#Links-to-Other-Notebooks